Rakefile

```diff
@@ -1,15 +1,20 @@
 require "bundler/gem_tasks"
 require "rake/testtask"
 require "standard/rake"
-require "rdoc/task"
+require "yard"
+require "yard/rake/yardoc_task"
 
-RDoc::Task.new do |rdoc|
-  rdoc.main = "docs/rdoc.rdoc"
-  rdoc.rdoc_files.include("docs/rdoc.rdoc",
-    "lib/sidekiq/api.rb",
-    "lib/sidekiq/client.rb",
-    "lib/sidekiq/worker.rb",
-    "lib/sidekiq/job.rb")
+YARD::Rake::YardocTask.new do |yard|
+  # keeping this in here for now, but code can be deleted once we are ready
+  # to push. .yardopts takes care of this.
```

Comment:
@mperham just leaving a comment here for you so it's not overlooked when the come comes to mergs :)